src/exceptions.h

```diff
@@ -65,6 +65,11 @@ class FileNotGoodException : public BaseException {
 	FileNotGoodException(const std::string &s): BaseException(s) {}
 };
 
+class DatabaseException : public BaseException {
```

Comment:
DatabaseException sounds better than FileNotGoodException. PostgreSQL is not a file.